test/python/test_ibmqjob_states.py

```diff
@@ -442,6 +459,60 @@ def get_job(self, job_id):
         raise ApiError()
 
 
+class ThrowingServerButJobFinishedAPI(BaseFakeAPI):
+    """Class for emulating an API throwing in the middle of execution, but
+    corresponding Job completes. This can simulate a server exception, but
+    not a a job problem, indeed, this will simulate a successfully completed
+    job.
+    """
+
+    _job_status = [
+        {'status': 'COMPLETED'}
+    ]
+
+    def get_status_job(self, job_id):
```

Comment:
I think you don't need to override this method for this specific API.

Reply:
Yep, I need it. `BaseFakeApi.get_status_job()` calls `get_job()`, which, btw, is not what happens in the real API.  Calling `get_job()` means, that, for example, `job.status()` will end up calling `get_job()` so throwing an ApiError, making impossible to test what I wanted here.

Reply:
You're right, I realize it later but forgot to remove this comment!